doc/plots.rst

```diff
@@ -0,0 +1,115 @@
+#####
+Plots
+#####
+
+The **Plots** pane shows the figures and images generated in the Editor, Console or the Variable Explorer during your session, allowing you to interact with them in several ways.
+
+.. image:: images/plots/plots-execution.gif
+   :align: center
+   :alt: Spyder main window showing execution for the Plots pane
+
+|
+
+The plots in the Plots pane depend on the Console you are working on. 
+This means that when switching to another console, the plots from your previous one will disappear, showing you a blank pane.
+
+.. image:: images/plots/plots-switch.gif
+   :align: center
+   :alt: Spyder's plots pane and console showing switching consoles
+
+|
+
+
+
+============
+Options Menu
+============
+
+The options menu in the top right of the :guilabel:`Plots` pane offers several options to customize your plots visualization.
+
+
+~~~~~~~~~~~~~~~~~~~~
+Mute inline plotting
+~~~~~~~~~~~~~~~~~~~~
+
+The :guilabel:`Mute inline plotting` option is enabled by default, preventing the figures from also appearing in the Console. 
+If you deactivate this option, the plots are generated will appear in the Console every time you run your code.
+
+.. image:: images/plots/plots-inline-plotting.gif
+   :align: center
+   :alt: Spyder's plots pane and console showing plots in the console
+
+|
+
+
+~~~~~~~~~~~~~~~~~
+Show plot outline
+~~~~~~~~~~~~~~~~~
+
+The :guilabel:`Show plot outline` option shows an outline box surrounding the area of the plot, with which you can export the plots.
+
+.. image:: images/plots/plots-outline.gif
+   :align: center
+   :alt: Spyder's plots pane showing a plot's outline
+
+|
+
+
+~~~~~~~~~~~~~~~~~~~
+Fit plots to window
+~~~~~~~~~~~~~~~~~~~
+
+The :guilabel:`Fit plots to Window` option is also enabled by default making the figures' size to match the size of the pane's window. 
+Disabling it will allow you to zoom the plots in or out using the Zoom buttons at the top of the pane.
```

Comment:
```suggestion
Disabling it will display plots at their native size, and allow you use the zoom buttons at the top of the pane to scale them manually.
```

* Improve descriptiveness, clarity and accuracy
* Reduce redundancy and fix style issues